shell/platform/darwin/macos/framework/Source/FlutterMetalRenderer.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+#import <Metal/Metal.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterEngine.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterView.h"
+#import "flutter/shell/platform/embedder/embedder.h"
+
+/**
+ * Provides the renderer config needed to initialize the embedder engine. This is initialized during
+ * FlutterEngine creation and then attached to the FlutterView once the FlutterViewController is
+ * initialized.
+ */
+@interface FlutterMetalRenderer : NSObject <FlutterTextureRegistry>
+
+@property(nonatomic, readonly, nonnull) id<MTLDevice> mtlDevice;
+
+@property(nonatomic, readonly, nonnull) id<MTLCommandQueue> mtlCommandQueue;
```

Comment:
It's extremely unusual to put the class prefix on a variable name. I don't see any reason for it here since there's nothing to confuse it with.

Reply:
Will revise the PR and remove all uses of prefixes. My default naming scheme is convert the type-name to var-name, which doesn't read well in this case :-)